pom.xml

```diff
@@ -41,6 +41,7 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<spring-cloud-release.version>Finchley.RC2</spring-cloud-release.version>
 		<spring-boot-release.version>2.0.2.RELEASE</spring-boot-release.version>
+		<zipkin-gcp.version>0.6.2</zipkin-gcp.version>
```

Comment:
when you feel ready. we can bump this to 1.0.0 ok. it is your call

Reply:
@adriancole Can you please comment on #749? I wonder if this requires changes to the extractor. Thanks!